integrations/sveltekit/.gitignore

```diff
@@ -0,0 +1,24 @@
+node_modules
+
+# Output
+.output
+.vercel
+.netlify
+.wrangler
+/.svelte-kit
+/build
+/dist
+
+# OS
+.DS_Store
+Thumbs.db
+
+# Env
+.env
+.env.*
+!.env.example
+!.env.test
+
+# Vite
+vite.config.js.timestamp-*
+vite.config.ts.timestamp-*
```

Comment:
that’s pretty much what’s in the root .gitignore anyway, I think we can safely remove this file here

Reply:
Added `.svelte-kit` to root `.gitignore` and removed this